homeassistant/components/go2rtc/server.py

```diff
@@ -112,6 +112,10 @@ async def _start(self) -> None:
             await self._stop()
             raise Go2RTCServerStartError from err
 
+        # Check the server version
+        client = Go2RtcRestClient(async_get_clientsession(self._hass), DEFAULT_URL)
+        await client.validate_server_version()
```

Comment:
Not needed as already done in the async_setup_entry

Reply:
Yes, but don't you think it makes sense to check it here too?